docs/source/index.rst

```diff
@@ -33,6 +33,7 @@ modularity and versatility for a broad set of industry use cases.
    architecture
    Fabric-FAQ
    CONTRIBUTING
+	 style_guide
```

Comment:
Could not find it in the TOC in the build? Will this be in the TOC in the contributing section or something that would could to be navigated to?